packages/editor-ui/src/components/CredentialEdit/CredentialEdit.vue

```diff
@@ -279,9 +279,16 @@ const requiredPropertiesFilled = computed(() => {
 });
 
 const credentialPermissions = computed(() => {
+	/**
+	 * Credentials don't have permissions until they have been created on the BE
+	 * credential:update allows the users modify the form fields at least until the credential is saved
+	 * credential:share allows the users to share not saved credentials
+	 */
 	return getResourcePermissions(
-		((credentialId.value ? currentCredential.value : credentialData.value) as ICredentialsResponse)
-			?.scopes,
+		(currentCredential.value as ICredentialsResponse)?.scopes ?? [
+			'credential:update',
+			'credential:share',
```

Comment:
These scopes should be best to get from the home project of the credential.
Maybe a user can create and update a credential but not share.
Until the credential is not saved our only chance is to check if the the user has the `credential:share` scope on the project in which they try to create the credential.

Reply:
i've changed it so it defaults to the project scopes until the credential is persisted